airflow/dags/payments_views/mst_transactions_gtfs_enhanced.sql

```diff
@@ -26,7 +26,7 @@ device_transactions AS (
         DATETIME(TIMESTAMP(transaction_date_time_utc), "America/Los_Angeles") as transaction_date_time_pacific,
         location_id,
         location_name,
-        route_id,
+        TRIM(route_id) as route_id,
```

Comment:
Do you know if we _need_ to trim the `route_id`? Even if we do, I think this trimming would be better further upstream (probably in `stg_cleaned_device_transactions`) rather than here -- in the GTFS we're considering a clean `route_id` to be one that is already trimmed, correct?

Reply:
Yeah fair, I don't think that there are any `route_id` values where it makes a difference right now but yes the data has been trimmed on the GTFS side (`dim_routes` has trimmed values). Happy to move that to `stg_cleaned_device_transactions` if that's the right place.

Reply:
That would be the right place. Go for it.

Reply:
Added in https://github.com/cal-itp/data-infra/pull/1233/commits/30bd391ddc996c8488aea90e88856cf4013c1479